ScanQR/Example/ViewController.swift

```diff
@@ -51,21 +45,25 @@ class ViewController: UIViewController {
     }
 }
 
-extension ViewController:ScanDelegate{
-    func didScanedCode(value: String) {
+extension ViewController: ScanDelegate{
```

Comment:
Make sure you're using SwiftLint so it makes you follow its coding conventions. aways have a space before `{`.